compiler/modules/nback/scaffold.bal

```diff
@@ -235,16 +243,16 @@ class Scaffold {
         return d:location(self.file, pos);
     }
 
-    function setDebugLocation(llvm:Builder builder, bir:Position pos, "file"? fileOnly = ()) {
+    function setDebugLocation(llvm:Builder builder, bir:Position pos, DebugLocationOrigin origin) {
         DISubprogram? diFunc = self.diFunc;
         if diFunc != () {
```

Comment:
Take advantage of narrowing after return. Use an `if` statement to identify all the cases where we do not need to set a debug location, and return immediately if we do not. Then use if to test whether it's just a file.

Reply:
fixed